### ISSUE TITLE
Able to run terraform plan without conftest

### ISSUE THREAD

**Issue by jarnohenneman** [cds-snc/terraform-plan#195]
Is it possible to run terraform plan without conftest?

```
# Setup Terraform, Terragrunt, and Conftest
- name: Setup terraform tools
  uses: cds-snc/terraform-tools-setup@v1
  env:
    #   CONFTEST_VERSION: 0.30.0
    TERRAFORM_VERSION: 1.4.6
    TERRAGRUNT_VERSION: 0.45.10
  #   TF_SUMMARIZE_VERSION: 0.2.3

- name: Terragrunt plan
  uses: cds-snc/terraform-plan@v3
  with:
    directory: ./terraform
    comment-delete: "true"
    github-token: "${{ secrets.GITHUB_TOKEN }}"
    terragrunt: "true"
```

Resulting in failing pipelines;

```
Plan changes

✅   Terraform Init: success
✅   Terraform Validate: success
✅   Terraform Format: success
✅   Terraform Plan: success
❌   Conftest: failed

Plan: 5 to add, 0 to change, 0 to destroy

Show summary
Show plan
Show Conftest results
```

**Comment by patheard** [comment]
Yup, it's possible thanks to #173:
```yaml
- name: Terragrunt plan
  uses: cds-snc/terraform-plan@v3
  with:
    directory: ./terraform
    comment-delete: "true"
    github-token: "${{ secrets.GITHUB_TOKEN }}"
    skip-conftest: "true"
    terragrunt: "true"
```

I'll get the docs updated with this change.

**Comment by jarnohenneman** [comment]
https://github.com/cds-snc/terraform-plan/blob/main/src/github.js#L77 is missing a check if conftest is true or not.

**Comment by patheard** [comment]
Looks to be fixed now by #197 with [`conftest` no longer running](https://github.com/cds-snc/terraform-plan/actions/runs/5075348607/jobs/9116467469?pr=197#step:5:125):
```
🧪 cat plan.json | tf-summarize -md

| CHANGE |      NAME      |
|--------|----------------|
| add    | `random_id.id` |

Deleting comment 'Test skip-conftest: 1562151849'
```

Must appreciated for the bug report and I'll aim to get a new release out tomorrow.

**Comment by patheard** [comment]
I just released [v3.0.6](https://github.com/cds-snc/terraform-plan/releases/tag/v3.0.6) of the action.  Hopefully it does the trick for you!

**Comment by jarnohenneman** [comment]
Thanks you @patheard . Appreciate the swift help/support.
